sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -468,6 +489,530 @@ def delete_ports_status_for_y_cable():
+
+        for i in range(1, 3):
+            mux_static_info_dict[("TOR_peer_Lane{}_Precursor1".format(i))] = cursor_tor1_values[i-1][0]
+            mux_static_info_dict[("TOR_peer_Lane{}_Precursor2".format(i))] = cursor_tor1_values[i-1][1]
+            mux_static_info_dict[("TOR_peer_Lane{}_Maincursor".format(i))] = cursor_tor1_values[i-1][2]
+            mux_static_info_dict[("TOR_peer_Lane{}_Postcursor1".format(i))] = cursor_tor1_values[i-1][3]
+            mux_static_info_dict[("TOR_peer_Lane{}_Postcursor2".format(i))] = cursor_tor1_values[i-1][4]
+
+    return mux_static_info_dict
+
+
+def post_port_mux_info_to_db(logical_port_name, table):
+
+    physical_port_list = logical_port_name_to_physical_port_list(logical_port_name)
+    if physical_port_list is None:
+        helper_logger.log_error("No physical ports found for logical port '{}'".format(logical_port_name))
+        return -1
+
+    if len(physical_port_list) > 1:
+        helper_logger.log_warning(
+            "Error: Retreived multiple ports for a Y cable port {}".format(logical_port_name))
+        return -1
+
+    for physical_port in physical_port_list:
+
+        if not _wrapper_get_presence(physical_port):
+            helper_logger.log_warning(
+                "Error: trying to post mux info without presence of port {}".format(logical_port_name))
+            continue
+
+        mux_info_dict = get_muxcable_info(physical_port, logical_port_name)
+        if mux_info_dict is not None:
+            #transceiver_dict[physical_port] = port_info_dict
+            fvs = swsscommon.FieldValuePairs(
+                [('tor_active',  mux_info_dict["tor_active"]),
+                 ('mux_direction',  str(mux_info_dict["mux_direction"])),
+                 ('manual_switch_count', str(mux_info_dict["manual_switch_count"])),
+                 ('auto_switch_count', str(mux_info_dict["auto_switch_count"])),
+                 ('link_status_self', mux_info_dict["link_status_self"]),
+                 ('link_status_peer', mux_info_dict["link_status_peer"]),
+                 ('link_status_nic', mux_info_dict["link_status_nic"]),
+                 ('nic_lane1_active', mux_info_dict["nic_lane1_active"]),
+                 ('nic_lane2_active', mux_info_dict["nic_lane2_active"]),
+                 ('nic_lane3_active', mux_info_dict["nic_lane3_active"]),
+                 ('nic_lane4_active', mux_info_dict["nic_lane4_active"]),
+                 ('Self_eye_height_lane1', str(mux_info_dict["Self_eye_height_lane1"])),
+                 ('Self_eye_height_lane2', str(mux_info_dict["Self_eye_height_lane2"])),
+                 ('Peer_eye_height_lane1', str(mux_info_dict["Peer_eye_height_lane1"])),
+                 ('Peer_eye_height_lane2', str(mux_info_dict["Peer_eye_height_lane1"])),
+                 ('NIC_eye_height_lane1', str(mux_info_dict["NIC_eye_height_lane1"])),
+                 ('NIC_eye_height_lane2', str(mux_info_dict["NIC_eye_height_lane2"])),
```

Comment:
Can we stay consistent with the use of case? Let's make the keys all lowercase, e.g., `self_eye_height_lane1`, `peer_eye_height_lane1`, `nic_eye_height_lane1` etc.

Reply:
fixed